docs/design/enforceHostNetwork.md

```diff
@@ -0,0 +1,31 @@
+# Enforcing host network design document
+
+By default, the ceph-csi controller plugin operates on the pod network
+but under some circumstances, like like setups with a dedicaated storage network, 
+where the pod network cannot connect to the ceph cluster,
+it is necessary to run the Ceph-CSI controller plugin pods on the host network.
+
+This document describes how the ceph-csi-operator can be configured to enforce the use of host network in 
+the Ceph-CSI controller plugin pods
+
+Enforcing of host network can be enabled in the operator configuration by setting `hostNetwork` to `true` in the `driverSpecDefaults.rcontrollerPlugin` section
```

Comment:
```suggestion
Enforcing of host network can be enabled in the operator configuration by setting `hostNetwork` to `true` in the `driverSpecDefaults.controllerPlugin` section
```

Reply:
fixed but not by directly committing this, but with additional context changes.